rcl_yaml_param_parser/test/correct_config.yaml

```
@@ -49,3 +49,7 @@ intel:
   ros__parameters:
     num_cores: 8
     arch: x86_64
+light:
+  ros__parameters:
+    is_on: yes
+    on_status: !!str yes
```

```suggestion
light:
  ros__parameters:
    is_on: yes
    on_status: !!str yes
    string_int: !!str 1234
    string_float: !!str 12.34
```

i think the bottom 2 should be recognized as string as well. it would be nice to add test?

---

Yes. I will add tests.